ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -440,6 +472,108 @@ export default {
         }
       },
     },
+    {
+      id: 'E12S Promise Chiseled Sculpture Tether',
+      netRegex: NetRegexes.tether({ target: 'Chiseled Sculpture', id: '0011' }),
+      durationSeconds: 20,
+      promise: async (data, matches) => {
+        // Calculate distance to center to determine inner vs outer
+        data.statues = data.statues || [];
+        if (data.statues.filter((s) => s.id === matches.targetId).length === 0) {
+          // Push statue to array immediately or else async causes the array
+          // to get duplicates of the statues
+          const statue = {
+            id: matches.targetId,
+          };
+          data.statues.push(statue);
+          const statueId = parseInt(matches.targetId, 16);
+          const statueData = await window.callOverlayHandler({
+            call: 'getCombatants',
+            ids: [statueId],
+          });
+
+          if (statueData === null || !statueData.combatants || !statueData.combatants.length)
+            return;
+
+          // Center = 0, -75
+          const x = statueData.combatants[0].PosX;
+          const y = statueData.combatants[0].PosY - -75;
+          statue.dist = Math.hypot(x, y);
+          if (data.statues.length > 1) {
+            // Sort so that closest statue (inner) is first
+            data.statues.sort((a, b) => a.dist - b.dist);
+          }
+        }
+      },
+      infoText: (data, matches, output) => {
+        // Player is the source of the tether for this one, oddly enough
+        if (data.me !== matches.source)
+          return;
+
+        if (data.statues.length > 0 && data.statues[0].id === matches.targetId)
+          data.statueDir = 'inner';
+        else if (data.statues.length > 1 && data.statues[1].id === matches.targetId)
+          data.statueDir = 'outer';
+        else
+          data.statueDir = 'unknown';
+
+        return output[data.statueDir]();
+      },
+      outputStrings: {
+        inner: {
+          en: 'Inner Statue',
+        },
+        outer: {
+          en: 'Outer Statue',
+        },
+        unknown: {
+          en: '???',
+        },
+      },
+    },
+    {
+      id: 'E12S Promise Palm Of Temperance SE',
+      netRegex: NetRegexes.startsUsing({ source: 'Guardian Of Eden', id: '58B4', capture: false }),
+      durationSeconds: 10,
+      infoText: (_1, _2, output) => output.knockback(),
+      outputStrings: {
+        knockback: {
+          en: 'Knockback SE',
+        },
+      },
+    },
+    {
+      id: 'E12S Promise Palm Of Temperance SW',
+      netRegex: NetRegexes.startsUsing({ source: 'Guardian Of Eden', id: '58B5', capture: false }),
+      durationSeconds: 10,
+      infoText: (_1, _2, output) => output.knockback(),
+      outputStrings: {
+        knockback: {
+          en: 'Knockback SW',
+        },
+      },
+    },
+    {
+      // Time the first dodge for lasers from the Rapturous Reach left/right split
```

Comment:
Do you think it's worth combining the haircut call with the #1 bait call? Mostly I'm trying to imagine what this would be like for tts users.

Reply:
That would require somehow suppressing the default haircut trigger for this mechanic specifically, which I'm not sure really makes sense?

Reply:
I was trying to suggest modifying the default haircut trigger.  To have it go "is this person the 1st bait, if so then call out "Left + Bait Inner" rather than having a separate trigger for the 1st bait, if that makes sense?

Reply:
The default haircut uses two separate triggers for left and right, which would both have to be modified for this or combined in some manner.

Reply:
I don't think modifying both sounds that bad, but if you think it'd be better separate, that's fine too.